includes/admin/class-wc-stripe-express-checkout-controller.php

```
@@ -46,9 +46,8 @@ public function admin_scripts(): void {
 
 		$stripe_settings = WC_Stripe_Helper::get_stripe_settings();
 		$params          = [
-			'key'            => WC_Stripe_Mode::is_test() ? $stripe_settings['test_publishable_key'] : $stripe_settings['publishable_key'],
-			'locale'         => WC_Stripe_Helper::convert_wc_locale_to_stripe_locale( get_locale() ),
-			'is_ece_enabled' => WC_Stripe_Feature_Flags::is_stripe_ece_enabled(),
+			'key'    => WC_Stripe_Mode::is_test() ? $stripe_settings['test_publishable_key'] : $stripe_settings['publishable_key'],
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Mode' in method 'admin_scripts'. (undefined)

(StaticAccess)